control/getCategories.js

```diff
@@ -1,12 +1,11 @@
-// const model = require('seed/lib/seed/errors/model')
 const database = require('../models/getdb.js')
 
-games = function(req, res) {
+gamesCategories = function(req, res) {
```

Comment:
You should never be initiating a variable with a loose declaration like this. Make sure we are always using const as the default and then changing to let if needed.